docs/topics/lombok.md

```diff
@@ -0,0 +1,137 @@
+[//]: # (title: Lombok compiler plugin)
+
+> The Lombok compiler plugin is [Experimental](components-stability.md). 
+> It may be dropped or changed at any time. Use it only for evaluation purposes. 
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-7112).
+>
+{type="warning"}
+
+The Lombok compiler plugin allows the generation and use of Java's Lombok declarations within the same mixed Java/Kotlin module.
```

Comment:
When you introduce the plugin, please define it as precisely as possible. In this case "The Lombok compiler plugin" has nothing that connects it with Kotlin. It may be obvious (like - we're in Kotlin docs, right?), but it's bad for SEO and may be unclear to those who come from Google.
You can say, for example, "Kotlin Lombok compiler plugin" or "Lombok plugin for Kotlin compiler", or ..?

Reply:
Ok, I'd erased it earlier but will return back😂